File: front-end/economycraft/lib/classes/company.dart
class Company {
  final int id;
  String name;
  String slogan;
  String avatarUrl;
  final int reputation;
  final double evaluation;
  bool isPublic;
  final int userId;
  final DateTime createdAt;
  final int lotNumber;
  final bool verified;

  Company({
    required this.id,
    required this.name,
    required this.slogan,
    required this.avatarUrl,
    required this.reputation,
    required this.evaluation,
    required this.isPublic,
    required this.userId,
    required this.createdAt,
    required this.lotNumber,
    required this.verified,
  });
}

File: code/server-processes/src/classes/classes/company.py
class Company:
    def __init__(self, id: int, created_at: str, name: str, reputation: int, is_public: bool, user_id: int, visibility_factor: int, ai: bool, notifications_enabled: bool):
        self.id = id
        self.created_at = created_at
        self.name = name
        self.reputation = reputation
        self.is_public = is_public
        self.user_id = user_id
        self.visibility_factor = visibility_factor
        self.ai = ai
        self.notifications_enabled = notifications_enabled